### PR TITLE
add twitterwall section

### PR DESCRIPTION
**Pull request by championswimmer**
Signed-off-by: Arnav Gupta <championswimmer@gmail.com>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -38,6 +38,8 @@ <h1 id="title"> {{ appCtrl.Event.name }} </h1>
         </md-content>
     </div>
 
+    <script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+"://platform.twitter.com/widgets.js";fjs.parentNode.insertBefore(js,fjs);}}(document,"script","twitter-wjs");</script>
```
Comment (creativepsyco) [inline index.html]: why does this needs to be here? I don't see the point. It be injected via another component.

Comment (championswimmer) [inline index.html]: ok yeah, i'll do so

File: assets/js/twitter-widget.js [hunk 2]
```diff
@@ -0,0 +1 @@
+!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+"://platform.twitter.com/widgets.js";fjs.parentNode.insertBefore(js,fjs);}}(document,"script","twitter-wjs");
```
Comment (championswimmer) [inline assets/js/twitter-widget.js]: @houndci this is a minified js. don't get all worked up :laughing:

**Comment by championswimmer** [comment]
@creativepsyco done. you can merge now :)

**Pull request merged**